docs/config.html

```diff
@@ -402,6 +402,44 @@ <h2 id="plugindefinition" class="title-2">[PluginDefinition]</h2>
   </pre>
 
   <p>This config value is then available to the build definitions as <code class="code">CONFIG.FOO.FOO_TOOL</code>.</p>
+
+  <ul class="bulleted-list">
+    <li>
+      <div>
+        <h3 class="mt1 f6 lh-title" id="plugindefinition.description">
+          Description <span class="normal">(string)</span>
+        </h3>
+
+        <p>
+        A description of the plugin. If set, this can be read and output by
+        <code class="code">plz help <plugin></code>.
```

Comment:
You might have to escape `<plugin>` since it's not a valid HTML tag - `&lt;plugin&gt;` should do.

You can also run `plz run //docs:server` and go to `localhost:8080` to verify the changes.